### ISSUE TITLE
Handy Tech Basic Braille 40: No Panning Commands

### ISSUE THREAD

**Issue by kjsapergia** [nvaccess/nvda#5806]
Hi,

I use a Handy Tech Basic Braille 40 display.
In trying various key combinations with NVDA, I've noticed that only keys 1 and 4 are used to move the display to a new line. That's fine, but I'm very surprised there doesn't seem to be a way to pan the display to the left or right if there's more text on a line. Would it be possible to implement this? It would definitely make reading material easier.

**Comment by josephsl** [comment]
Hi, two keys should be set aside for this purpose (we need to talk to the manufacturer about which keys are appropriate for panning the display). Thanks.

**Comment by bramd** [comment]
I was considering extending the key bindings for the Handy Tech driver already for other models. Given the driver design, your keys should be recognized by NVDA, however they are not mapped to the proper function yet. Could you please use the input gestures dialog to make your desired mapping? You'll find this in NVDA menu/preferences/input gestures.

Please post the key names you mapped to the panning functions in this issue so I can put them as defaults in the driver.

**Comment by kjsapergia** [comment]
Hi Bram,

Thanks for the tip. I didn't know you could use Input Gestures for that. Anyway, for the Basic Braille, I've assigned keys 2 and 5 (or B3 and B6 according to NVDA) to scroll back and forward respectively. I haven't assigned any other functions yet, but may do so in future.

Thanks again.

Yours Sincerely,
Kelly John Sapergia
Show Host and Production Director
The Global Voice Internet Radio
http://www.theglobalvoice.info

Personal Website: http://www.ksapergia.net
Business Website (KJS Productions): http://www.kjsproductions.com
Follow me on Twitter at: kjsapergia

**Comment by bramd** [comment]
@kjsapergia Thanks. B3 and B6 make sense as mapping for other Handy Tech displays as well, so I will add them to the driver soon for all models.

**Comment by leonardder** [comment]
CC @FelixGruetzmacher
